plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenArchetypeProjectWizard.java

```diff
@@ -52,7 +52,7 @@ public void addPages() {
       this.addPage(page);
```

Comment:
I think `MavenArchtypeProjectWizard` also needs the `cloudSdkExists()` handling as per `StandardProjectWizard`.